test/e2e/tests/swaps/swap-erc20-with-loaded-state.spec.ts

```diff
@@ -75,124 +79,235 @@ async function mockSwapQuotes(mockServer: MockttpServer) {
-              ratio: 0.9997007391742396,
-              alculationError: '',
-              bucket: 'low',
-              sourceAmountInUSD: 20049.4,
-              destinationAmountInUSD: 20055.4,
-              sourceAmountInNativeCurrency: 9.996738094827068,
-              destinationAmountInNativeCurrency: 10,
-              sourceAmountInETH: 9.996738094827068,
-              destinationAmountInETH: 10,
+        json: {
+          active: true,
+          networkId: 1,
+          chainId: 1,
+          chainName: 'Ethereum Mainnet',
+          nativeCurrency: {
+            name: 'Ether',
+            symbol: 'ETH',
+            decimals: 18,
+            address: ETH_ADDRESS,
+          },
+          iconUrl: 'https://s3.amazonaws.com/airswap-token-images/ETH.png',
+          blockExplorerUrl: 'https://etherscan.io',
+          networkType: 'L1',
+          aggregators: [
+            'airswapV3',
+            'airswapV4',
+            'oneInchV4',
+            'oneInchV5',
+            'paraswap',
+            'pmm',
+            'zeroEx',
+            'openOcean',
+            'hashFlow',
+            'wrappedNative',
+            'kyberSwap',
+            'airSwapV4_3',
+            'hashFlowV3',
+          ],
+          refreshRates: {
+            quotes: 30,
+            quotesPrefetching: 30,
+            stxGetTransactions: 10,
+            stxBatchStatus: 1,
+            stxStatusDeadline: 160,
+            stxMaxFeeMultiplier: 2,
+          },
+          parameters: {
+            refreshRates: {
+              quotes: 30,
+              quotesPrefetching: 30,
+              stxGetTransactions: 10,
+              stxBatchStatus: 1,
             },
+            stxStatusDeadline: 160,
+            stxMaxFeeMultiplier: 2,
           },
-        ],
+        },
       })),
   ];
 }
 
 describe('Swap', function () {
-  it('should swap WETH to ETH', async function () {
-    await withFixtures(
-      {
-        fixtures: new FixtureBuilder()
-          .withNetworkControllerOnMainnet()
-          .withTokensController({
-            allTokens: {
-              '0x1': {
-                '0x5cfe73b6021e818b776b421b1c4db2474086a7e1': [
-                  {
-                    address: '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
-                    symbol: 'WETH',
-                    decimals: 18,
-                    isERC721: false,
-                    aggregators: [],
-                  },
-                ],
+  const swapTestCases = [
+    // {
+    //   name: 'should swap WETH to ETH',
+    //   sourceToken: 'WETH',
+    //   destinationToken: 'Ether',
+    //   sourceAmount: '10',
+    //   expectedWethBalance: '40',
+    //   expectedEthBalance: '34.99991',
+    //   dismissWarning: false,
+    // },
+    {
+      name: 'should swap ETH to WETH',
+      sourceToken: 'Ethereum',
+      destinationToken: 'WETH',
+      sourceAmount: '10',
+      expectedWethBalance: '60',
+      expectedEthBalance: '14.99992',
```

Comment:
the spec is failing as this value is incorrect

```suggestion
      expectedEthBalance: '14.99993',
```

Reply:
same than before... just wondering if we can ignore this or make it like ~ a value, like balance ~ 14.999